welkin/models/encounter.py

```diff
@@ -3,6 +3,58 @@
 from welkin.models.base import Collection, Resource
 
 
+class Assessment(Resource):
+    def create(self):
+        return super().post(
+            f"{self._client.instance}/patients/{self.patientId}/encounters/{self._parent.id}/assessments"
+        )
+
+    def get(self):
+        return super().get(
+            f"{self._client.instance}/patients/{self.patientId}/encounters/{self._parent.id}/assessments/{self.id}"
+        )
+
+    def update(self, **kwargs):
+        return super().patch(
+            f"{self._client.instance}/patients/{self.patientId}/encounters/{self._parent.id}/assessments/{self.id}",
+            kwargs,
+        )
+
+    def delete(self):
+        return super().delete(
+            f"{self._client.instance}/patients/{self.patientId}/encounters/{self._parent.id}/assessments/{self.id}"
+        )
+
+    @property
+    def patientId(self):
+        if self._parent._parent:
```

Comment:
Should probably be testing for type here, it isn't clear what `self._parent._parent` is. `if isinstance(self._parent, Patient)`?